geometry_msgs/Point point
geometry_msgs/Quaternion orientation
---
int8 retCode